src/SixLabors.Fonts/Buffer{T}.cs

```diff
@@ -26,7 +26,7 @@ public Buffer(int length)
         this.buffer = ArrayPool<byte>.Shared.Rent(bufferSizeInBytes);
         this.length = length;
 
-        ByteMemoryManager<T> manager = new(this.buffer);
+        using ByteMemoryManager<T> manager = new(this.buffer);
```

Comment:
`Dispose()` doesn't actually do anything for this type but given that is an implementation detail disposing is probably appropriate. 

However, the manager should be held as a private field of the parent `Buffer<T>` and disposed at the end of the parent lifecycle.

Reply:
Sorry, didn't get the idea about the purpose of private field. You can contribute to my repo if you wish.